### PR TITLE
docs suggestion - small revision in RunExamples.md

### PR DESCRIPTION
**Pull request by hangtwenty**

### CODE REVIEW

File: docs/.vitepress/theme/components/RunExamples.md [hunk 1]
```diff
@@ -1,7 +1,22 @@
...
 pnpm run examples
 ```
 
+
+<details>
+    <summary>Make sure you run <strong>pnpx build</strong> first</summary>
+
+```shell
+cd vite-plugin-pwa
+pnpm run install  # (if you need to)
```
Comment (userquin) [inline docs/.vitepress/theme/components/RunExamples.md:14]: we are installing project dependencies, should be: `pnpm install`

File: docs/.vitepress/theme/components/RunExamples.md [hunk 2]
```diff
@@ -1,7 +1,22 @@
...
+pnpm run build
+pnpm run examples
+```
+
+If you don't do `pnpx run build` first,
```
Comment (userquin) [inline docs/.vitepress/theme/components/RunExamples.md:19]: should be: `pnpm run build`

Comment (hangtwenty) [reply docs/.vitepress/theme/components/RunExamples.md:19]: Doh! I meant to put `pnpm run build`, didn't meant to put `pnpx`, thanks for catching that!

**Comment by hangtwenty** [comment]
Thanks for the review @userquin!

File: docs/.vitepress/theme/components/RunExamples.md [hunk 3]
```diff
@@ -1,7 +1,22 @@
...
 pnpm run examples
 ```
 
+
+<details>
+    <summary>Make sure you run <strong>pnpx build</strong> first</summary>
+
+```shell
+cd vite-plugin-pwa
+pnpm run install
```
Comment (userquin) [inline docs/.vitepress/theme/components/RunExamples.md:14]: should be `pnpm install`

**Review by userquin — APPROVED** [review]

**Pull request merged**